pipelines/optimus/example_test_outputs.json

```diff
@@ -2,23 +2,23 @@
   "outputs": {
     "Optimus.tag_gene_exon_log": [
       [
-        "gs://broad-dsde-mint-dev-cromwell-execution/cromwell-executions/Optimus/c98322d7-406d-4e3f-96c8-e2bd1429e645/call-AlignTagCorrectUmis/shard-0/AlignTagCorrectUmis/d250b98d-a2f0-4b19-be86-e347bd076a5f/call-TagGeneExon/shard-0/gene_exon_tag_summary.log"
+        "gs://hca-dcp-mint-test-data/10x/demo/cromwell-executions/Optimus/c98322d7-406d-4e3f-96c8-e2bd1429e645/call-AlignTagCorrectUmis/shard-0/AlignTagCorrectUmis/d250b98d-a2f0-4b19-be86-e347bd076a5f/call-TagGeneExon/shard-0/gene_exon_tag_summary.log"
```

Comment:
I guess this is fine to do find/replace, but in the example that was given, they point to real files that a mint dev could inspect, whereas the changed files don't exist. 

Other opinions on this @samanehsan? I guess for public users the existence of these files is a bit pointless since they can't access them anyway (unless we moved them the `hca-dcp-mint-test-data` bucket

Reply:
I actually copied over those output files to the public bucket. Are you sure they're not there? I may have missed some or given the wrong paths here.

Reply:
Oops, my fault! Sorry. You're right.